nuxeo-drive-core/src/test/java/org/nuxeo/drive/fixtures/DefaultFileSystemItemFactoryFixture.java

```diff
@@ -1054,6 +1052,69 @@ public void testScrollDescendantsIncludingCollections() {
         assertEquals("testFolder", descendant.getName());
     }
 
+    /**
+     * Tests the following hierarchy:
+     *
+     * <pre>
+     * syncRoot             Synchronization root for joe with Read access
+     * |-- folder           Blocked inheritance
+     * |   |-- placeless    Read access for joe (placeless document)
+     * |
+     * |-- descendant1      Read access for joe (inheritance)
+     * |-- descendant2      Read access for joe (inheritance)
+     * |-- ...              Read access for joe (inheritance)
+     * </pre>
+     *
+     * syncRoot should be synchronized with its directly accessible descendants to obtain the following hierarchy
+     * client-side:
+     *
+     * <pre>
+     * syncRoot
+     * |-- descendant1
+     * |-- descendant2
+     * |-- ...
+     * </pre>
+     *
+     * The placeless document will not be synchronized.
+     *
+     * @since 10.3
+     */
+    @Test
+    public void testScrollDescendantsWithBlockedInheritance() {
+        log.trace("Create \"placessDocument\" in \"/syncRoot/folder\"");
+        DocumentModel placeless = session.createDocumentModel(folder.getPathAsString(), "placess", "File");
```

Comment:
Nitpick: placess -> placeless (here and the line above)

Reply:
Nice catch